data_splitting.py

```diff
@@ -14,38 +14,41 @@
 
 from node import Node
 
-
-def process_data_splitting_scenario(x_train, y_train, x_test, y_test, dataset_name,
-                                    nodes_count, amounts_per_node,
-                                    overlap_or_distinct, testset_option):
+# TODO this method should propably be in the Scenario class
+def process_data_splitting_scenario(scenario):
     """Return a list of nodes populated with their train and test data (not pre-processed)"""
 
     #%% Fetch parameters of scenario
     
+    x_train = scenario.x_train
+    y_train = scenario.y_train
+    x_test = scenario.x_test
+    y_test = scenario.y_test
+    
     # Describe data
-    print('\n### Data loaded: ', dataset_name)
-    print('- ' + str(len(x_train)) + ' train data with ' + str(len(y_train)) + ' labels')
+    print('\n### Data loaded: ', scenario.dataset_name)
+    print('- ' + str(len(scenario.x_train)) + ' train data with ' + str(len(y_train)) + ' labels')
```

Comment:
`y_train` should be prefixed with `scenario.` too

Reply:
will do